tests_e2e/orchestrator/lib/agent_test_suite.py

```diff
@@ -84,15 +111,23 @@ def _set_context(self, node: Node):
                 private_key_file=connection_info['private_key_file'],
                 ssh_port=connection_info['port']))
 
+        self.__context.log = log
         self.__context.node = node
-        self.__context.suite_name = f"{self._metadata.full_name}:{runbook.marketplace.offer}-{runbook.marketplace.sku}"
+        self.__context.suite_name = f"{self._metadata.full_name}_{runbook.marketplace.offer}-{runbook.marketplace.sku}"
 
     @property
     def context(self):
         if self.__context is None:
             raise Exception("The context for the AgentTestSuite has not been initialized")
         return self.__context
 
+    @property
+    def _log(self) -> Logger:
+        """
+        Returns a reference to the LISA Logger.
+        """
+        return self.context.log
+
```

Comment:
self._log is the LISA Logger, while agent_test_logger is the global test Logger (which redirects output to the log file for each suite)